.gitignore

```diff
@@ -23,7 +23,7 @@ docs/_site/
 
 # Theme files
 /theme/
-!/theme/mocha.dark.json
+!/theme/mocha dark.json
```

Comment:
Since the file is deleted, I think we should just remove this line from the gitignore as well.